test/src/HoistedNodeModuleTest.ts

```diff
@@ -317,3 +317,28 @@ test.ifAll("npm tar", () =>
     }
   )
 )
+
+//github.com/electron-userland/electron-builder/issues/8881
+test.ifAll("pnpm node-linker=hoisted", () =>
```

Comment:
We need to add more test cases for pnpm hoisted config.

![image](https://github.com/user-attachments/assets/76eadeda-872e-4284-bf43-fdb4c3356ac5)

Reply:
Under these two configurations, we can still use `pnpm list` instead of `npm list`.

Reply:
Will do!